docs/apache-airflow-providers-google/operators/cloud/cloud_composer.rst

```diff
@@ -15,7 +15,7 @@
     specific language governing permissions and limitations
     under the License.
 
-Google Cloud Dataproc Metastore Operators
+Google Cloud Composer Operators
 =========================================
```

Comment:
It is not required.

Reply:
Now it's aligned.

Reply:
I remember seeing complaints about it @mik-laj  (and for sure my IDE complains)

Reply:
All right. Found it (you were right @mik-laj - it might be longer (but not shorter):

```
In order to write a title, you can either underline it or under and overline it. The following examples are correct titles.

*****
Title
*****

subtitle
########

subsubtitle
**********************
and so on
Two rules:

* If under and overline are used, their length must be identical
* The length of the underline must be at least as long as the title itself
```

https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings

Reply:
Very stupid and confusing rule BTW (from maintenance POV). 

Why when you make a longer text you have to adapt it, but when you make it shorter, you don't ? 

:shrug: :smile: